packages/core/template/project/Testing.md

```diff
@@ -0,0 +1,46 @@
+---
+  layout: default.md
+  title: "Testing guide"
```

Comment:
Again, if you intend to have these pages please make them more accessible / findable directly from sitenav